democracylab/emails.py

```diff
@@ -15,9 +15,12 @@
 
 class EmailSection(Enum):
     Header = 'Header'
+    Headerleft = 'Headerleft'
```

Comment:
nit: Let's try to keep the casing consistent 
```suggestion
    Header_Left= 'Header_Left'
```